qt/scientific_interfaces/Direct/ALFView_model.cpp

```diff
@@ -0,0 +1,124 @@
+#include "ALFView_model.h"
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+//     NScD Oak Ridge National Laboratory, European Spallation Source
+//     & Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+
+#include "MantidAPI/Algorithm.h"
+#include "MantidAPI/AlgorithmManager.h"
+#include "MantidAPI/AnalysisDataService.h"
+#include "MantidAPI/MatrixWorkspace.h"
+#include "MantidAPI/NumericAxis.h"
+
+#include "MantidGeometry/Instrument.h"
+
+#include "MantidKernel/Unit.h"
+
+#include <utility>
+
+namespace {
+const std::string tmpName = "ALF_tmp";
```

Comment:
Can we use the standard uppercase name for file-level statics? It makes it easier to know where they are defined when reading the code.